css/style.css

```diff
@@ -0,0 +1,27 @@
+
+html{
+
+    box-sizing: border-box;
+}
+
+*, *::before, *::after{
+
+    box-sizing: inherit;
+}
+
+body { 
+
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    box-sizing: border-box;
+}
+
+
+.header__logo{
+
+    font-family: 'PT Serif', serif;
+    font-weight: bold;
+    font-size: 28px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/miniProject/pullRequest?prid=5576209)